index.md

```diff
@@ -1,213 +1,270 @@
 ---
-title: Welcome to Flutter
-sidebar: home_sidebar
+title: Easy and fast SDK for beautiful apps
 layout: page
+hide_title: true
 ---
 
-{% include note.html content="_Flutter is an early-stage open-source project_. We hope you try Flutter out and send us [feedback](#get-support)." %}
+<div class="homepage">
+
+    <section class="homepage__key_points">
+        <h1 class="homepage__title">
+        Flutter makes it easy
+        and fast to build
+        beautiful mobile apps.
+        </h1>
+
+        <div class="homepage__tagline">
+        Flutter is a new mobile app SDK
+        to help
+        developers and designers build
+        modern mobile apps for iOS and Android.
+        </div>
+
+        <div class="homepage__button_row">
+        <a href="/getting-started/" class="get-started-button">Get Started</a>
+        {% include note.html content="_Flutter is an alpha, open-source project_." %}
+        </div>
+
+        <div class="key-points">
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Deliver features faster</div>
+
+                <p>
+                Refresh times so fast, you can "paint" your app to life
+                on hardware, emulators, and simulators.
+                </p>
+            </div>
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Craft beautiful UIs</div>
+
+                <p>
+                Delight your users and make your brand stand out with
+                rich motion, smooth scrolling, and
+                beautiful customizable components.
+                </p>
+            </div>
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Used by Google</div>
+
+                <p>
+                Flutter is used by Google, works with Firebase
+                and other mobile app SDKs,
+                and is open source.
+                </p>
+            </div>
+
+        </div>
+    </section>
+
+    <section class="homepage__hot_reload">
+        <h1>Fast development</h1>
+
+        <p>
+            Flutter's <em>hot reload</em> helps you quickly
+            and easily experiment, build UIs, add features, and fix
+            bug faster. Experience sub-second reload times,
+            without losing state, on
+            emulators, simulators, and hardware for iOS
+            and Android.
+        </p>
+
+        <div class="hot-reload-gif-container">
+            <img src="/images/intellij/hot-reload.gif" class="hot-reload-gif" alt="Make a change in your code, and your app is changed instantly.">
+        </div>
+    </section>
+
+    <section class="homepage__beautiful_uis">
+        <h1>Expressive, beautiful UIs</h1>
+
+        <p>
+        Delight your users with Flutter's built-in
+        beautiful Material Design and
+        Cupertino (iOS-flavor) widgets, rich motion APIs,
```

Comment:
Can we have GIF or embedded video to show "motion" and smooth scrolling? At the minimum, a link to a video demo on YouTube would help support our claims.